src/modules/conversations.ts

```diff
@@ -29,12 +26,13 @@ class ConversationsApi extends BaseApi {
      *         * quality:calibration:view
      *         * quality:evaluation:editScore
      *
-     * @example
```

Comment:
I noticed you pulled out most of the @example tags but in playing on the tsdoc playground it seems that @example is supported and renders out `Example N`: blocks.

Did you have issues transforming the example blocks to markdown?  Just curious.

Reply:
Typedoc doesn't support **@example** tags and just renders them like any other tag that it comes across (lowercase bolded text). Their docs suggest just using code blocks as examples. https://typedoc.org/guides/doccomments/#supported-tags. Not that we have to follow that, but I felt that the extra "**example**" text just looked off and detracted from the docs when using the tag in conjunction with the code blocks. We can also discuss this further if you'd like.

Reply:
ah gotcha.  Was looking at tsdoc vs typedoc.

Yeah, if it's not formally supported and doesn't provide us a structured block for configuration/styling, I'm def good with just using code blocks.